responsibleai/responsibleai/rai_insights/rai_insights.py

```diff
@@ -423,6 +425,14 @@ def _get_dataset(self):
                 msg = "Model does not support predict method for given"
                 "dataset type"
                 raise ValueError(msg) from ex
+            post_pred_features = dataset.columns
```

Comment:
We should not address error scenarios here but raise an error in the input validations for rai_insights. It really complicates our package design. 

The customer should wrap there models to adhere to our package.

Reply:
done, created a separate PR: https://github.com/microsoft/responsible-ai-toolbox/pull/1055